pythonx/jedi_vim.py

```diff
@@ -222,7 +222,12 @@ def get_project():
         "get(b:, 'jedi_environment_path', g:jedi#environment_path)"
     )
     vim_project_path = vim_eval("g:jedi#project_path")
-    vim_added_sys_path = vim_eval("g:jedi#added_sys_path")
+    vim_added_sys_path = vim_eval(
+        "get(g:, 'jedi#added_sys_path', [])"
+    )
+    vim_added_sys_path += vim_eval(
+        "get(b:, 'jedi_added_sys_path', [])"
+    )
```

Comment:
There's a bug in the logic below that gets even worse when we start using buffer definitions.

The problem is that `vim_added_sys_path` is not part of the `cache_key`. Would you agree? I feel like we should add it there as well.

Reply:
@davidhalter I agree, David. Please take a look at the latest commit. I have just added 'g:jedi#added_sys_path' and 'b:jedi_added_sys_path' to 'cache_key'.